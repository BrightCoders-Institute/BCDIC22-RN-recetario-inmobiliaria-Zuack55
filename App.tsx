import React, { Component } from 'react'
import {  } from 'react-native';
import ScrollViewExample from './components/Cards'

export class App extends Component {
  render() {
    return (
      <ScrollViewExample />
    )
  }
}

export default App