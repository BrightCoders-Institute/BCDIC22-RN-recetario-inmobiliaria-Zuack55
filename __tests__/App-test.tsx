import React from "react";
import {create} from 'react-test-renderer';
import App from "../App";

const testRenderer = create(<App />);

test('App component renders correctly', () => {
    expect(testRenderer).toMatchSnapshot();
});

test('App component renders correctly loeader', () => {
    const testInstance = testRenderer.root;
    expect(testInstance.findByProps({testID: 'loader'})).toBeTruthy();
});
