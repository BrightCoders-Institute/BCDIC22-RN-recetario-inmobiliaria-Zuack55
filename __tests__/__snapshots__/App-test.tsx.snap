// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders correctly 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
  testID="loader"
>
  <ActivityIndicator
    color="#00B074"
    size={70}
    style={
      {
        "marginTop": 300,
      }
    }
  />
</View>
`;
